README.md

```diff
@@ -103,75 +76,27 @@ class YatLib {
-            val config = YatAppConfig(
-                name = "Super Cool Wallet",
-                sourceName = "SCW",
-                // used in partner-specific URLs
-                pathKey = "scw",
-                // app-specific public key
-                pubKey = "{64 character hex public key}",
-                // app-specific access code
-                code = "66b6a5aa-11b4-12a9-1c1e-84765ef174ab",
-                // app-specific authentication token
-                authToken = "AppToken84765783"
-            )
-            // Yat records
-            val yatRecords = listOf(
-                YatRecord(
-                    YatRecordType.ADA_ADDRESS,
-                    "DdzFFzCqrhsgwQmeWNBTsG8VjYunBLK9GNR93GSLTGj1FeMm8kFoby2cTHxEHBEraHQXmgTtFGz7fThjDRNNvwzcaw6fQdkYySBneRas"
-                ),
-                YatRecord(
-                    YatRecordType.DOT_ADDRESS,
-                    "GC8fuEZG4E5epGf5KGXtcDfvrc6HXE7GJ5YnbiqSpqdQYLg"
-                ),
-                YatRecord(
-                    YatRecordType.BTC_ADDRESS,
-                    "1NDyJtNTjmwk5xPNhjgAMu4HDHigtobu1s"
-                ),
-                YatRecord(
-                    YatRecordType.ETH_ADDRESS,
-                    "108dEFa0272dC118EF03a7993e4fC7A8AcF3a3d1"
-                ),
-                YatRecord(
-                    YatRecordType.XTR_ADDRESS,
-                    "d2e4db6dac593a9af36987a35676838ede4f69684ba433baeed68bce048e111b"
-                ),
-                YatRecord(
-                    YatRecordType.XMR_STANDARD_ADDRESS,
-                    "4AdUndXHHZ6cfufTMvppY6JwXNouMBzSkbLYfpAV5Usx3skxNgYeYTRj5UzqtReoS44qo9mtmXCqY45DJ852K5Jv2684Rge"
-                )
+   
+            val config = YatConfiguration(
+                appReturnLink = "app://y.at?action", // deep link to return to the app
+                organizationName = "Yat Labs",
+                organizationKey = "yat",
             )
-            // initialize the library
-            YatLib.initialize(
+   
+            YatIntegration.setup(
+                context = this,
                 config = config,
-                /*
-                * User information below is utilized to link the partner application user
-                * to the Yat user.
-                */
-                userId = "partner_app_user_identifier",
-                userPassword = "partner_app_user_password",
-                /*
-                * YatLib.ColorMode.LIGHT for light mode,
-                * YatLib.ColorMode.DARK for dark mode,
-                */
-                colorMode = YatLib.ColorMode.LIGHT,
-                /*
-                * Delegate interface is described below.
-                */
+                colorMode = YatIntegration.ColorMode.LIGHT, // Choose between LIGHT and DARK
                 delegate = this,
-                yatRecords = yatRecords
+                environment = YatIntegration.Environment.SandBox, // Choose between SandBox and Production
```

Comment:
User can configure the framework to connect with his own server. So there is more options than only sandbox and production.